freud/_order.pxd

```diff
@@ -46,28 +46,26 @@ cdef extern from "NematicOrderParameter.h" namespace "freud::order":
         vec3[float] getNematicDirector()
 
 
-cdef extern from "HexOrderParameter.h" namespace "freud::order":
+cdef extern from "OrderParameter.h" namespace "freud::order":
     cdef cppclass HexOrderParameter:
-        HexOrderParameter(float, unsigned int, unsigned int)
+        HexOrderParameter(unsigned int)
         const freud._box.Box & getBox() const
-        void compute(freud._box.Box &,
-                     const freud._locality.NeighborList*,
-                     const vec3[float]*,
-                     unsigned int) nogil except +
+        void compute(const freud._locality.NeighborList*,
+                     const freud._locality.NeighborQuery*,
+                     freud._locality.QueryArgs) nogil except +
         # unsure how to pass back the std::complex,
         # but this seems to compile...
         shared_ptr[float complex] getPsi()
         unsigned int getNP()
         unsigned int getK()
 
-cdef extern from "TransOrderParameter.h" namespace "freud::order":
+cdef extern from "OrderParameter.h" namespace "freud::order":
```

Comment:
You don't need this line twice, as long as you keep the indent level the previous `cdef extern` just continues.